### DISCUSSION TITLE
Clearable on a select causes the tab to focus to the clear icon instead of the next input

### DISCUSSION THREAD

**Discussion by tjmcdonough** [mantinedev/mantine#4710] [Feedback]
When the 'clearable' feature is enabled for a select input and I use the Tab key, the expected behaviour is for the focus to shift to the next input field. However, the focus is currently moving to the 'clearable' icon within the select. Can we discuss possible solutions or adjustments for this?

**Comment by JustCreateDeveloper** [comment]
+1 Not user friendly the user has to tab twice to move to the next input
